src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -359,8 +359,30 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
   )
 
   banner(
-    s""" Usage: ${GoVerifier.name} -i <input-files> [OPTIONS] OR
-       |  ${GoVerifier.name} -p <directory> [OPTIONS]
+    s""" ${GoVerifier.name} supports three modes for specifying input files that should be verified:
+       |  ${GoVerifier.name} -i <input-files> [OPTIONS] OR
```

Comment:
to my understanding the project root is relevant to resolve Go's package imports. Why is there no optional project root for `-i`. Also, if the project root works the same for any input kind, then maybe the project root can be explained as a separate entity.

Reply:
This is what I tried to clarify on line 383:
You have the option `--include` in mind, which is indeed applicable to all 3 modes.
However, `--projectRoot` specifies a root directory based on which a unique identifier for each package should be derived (this is imho the only use case). This unique identifier is then used for caching purposes and the idea was that the same project on different computers (and thus different absolute paths) come up with the same unique package identifier (since the paths relative to the project root should be the same if configured correctly), such that cache files can be shared between machines

Reply:
Getting back to your original question, I honestly don't recall why we explicitly excluded the project root option from `-i`. It might be related to the fact that the provided files might not fully represent a package (e.g. the files are located in different folders but all have the same package clause or only a subset of files of a package has been provided)

Reply:
In the explanation, I believe it is still a good idea to give a separate explanation for `--projectRoot` if its semantics can be explained independent of the mode. Currently, it is also not clear to me what kind of criteria the argument to `--projectRoot` has to satisfy.